pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
@klopfdreh 
JSONFunction are rendered as regular strings, surrounded by double quotes. But I think the culprit is here:

https://github.com/tdunning/open-json/blob/master/src/main/java/org/json/JSONStringer.java#L261

We should rename JsonFunction back to JSONFunction